src/timeline.js

```diff
@@ -42,19 +42,27 @@ export default class Timeline extends React.Component {
   static TIMELINE_MODES = Object.freeze({
     SELECT: 1,
     DRAG: 2,
-    RESIZE: 4,
+    RESIZE: 4
   });
 
   static propTypes = {
     items: PropTypes.arrayOf(PropTypes.object).isRequired,
     groups: PropTypes.arrayOf(PropTypes.object).isRequired,
+    // groupOffset is the width of the column when multipleColumns is false or the default width of the columns
+    // if column.width is not configured.
     groupOffset: PropTypes.number.isRequired,
+    multipleColumns: PropTypes.bool, // If the timeline should have multiple columns or only one column.
+    dataGridColumns: PropTypes.arrayOf(
+      PropTypes.shape({
+        headerKey: PropTypes.number.isRequired
```

Comment:
I didn't yet read the answer to my question regarding what these definitions do. But I think things are missing, compared to what I've seen in the demo.

I'd rename this: `tableColumns`. I think it's more universal than "data grid".